variants/STM32C0xx/C011D6Y_C011F(4-6)(P-U)_C031F(4-6)P/variant_generic.cpp

```diff
@@ -57,4 +58,48 @@ const uint32_t analogInputPin[] = {
   12  // A12, PA14
 };
 
+// ----------------------------------------------------------------------------
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+  * @brief  System Clock Configuration
+  * @param  None
+  * @retval None
+  */
+// Removed the "WEAK" symbol as some of the following settings were overwritten
```

Comment:
This file is automatically generated. The generic `SystemClock_Config `have to be added in the `generic_clock.c` file.